src/components/application_manager/test/commands/hmi/get_urls_test.cc

```diff
@@ -234,7 +235,137 @@ TEST_F(GetUrlsTest, ProcessPolicyServiceURLs_SUCCESS) {
             (*command_msg)[am::strings::msg_params][am::hmi_response::urls][0]
                           [strings::url].asString());
 }
+TEST_F(GetUrlsTest, ProcessPolicyServiceURLs_IncorrectIdForSending_UNSUCCESS) {
+  MessageSharedPtr command_msg(
+      CreateMessage(NsSmartDeviceLink::NsSmartObjects::SmartType_Map));
+  (*command_msg)[am::strings::msg_params][am::strings::number] = "123";
+  (*command_msg)[am::strings::params][am::strings::connection_key] =
+      kConnectionKey;
+  (*command_msg)[am::strings::msg_params][am::hmi_request::service] =
+      kPolicyService;
+
+  ON_CALL(mock_app_manager_, event_dispatcher())
+      .WillByDefault(ReturnRef(mock_event_dispatcher_));
+
+  RequestFromHMIPtr command(CreateCommand<GetUrls>(command_msg));
+
+  EXPECT_CALL(mock_app_manager_, GetPolicyHandler()).Times(3);
+  EXPECT_CALL(policy_handler_, PolicyEnabled()).WillOnce(Return(true));
```

Comment:
@dcherniev `policy_handler_` shiuld be renamed to `mock_policy_handler_` as you are using mock class instead of real policy handler object